cloud/linode/linode.py

```diff
@@ -37,12 +37,77 @@
      - To keep sanity on the Linode Web Console, name is prepended with LinodeID_
     default: null
     type: string
+  displaygroup:
+    description:
+     - Add the instance to a Display Group in Linode Manager (alphanumeric, dashes, underscore)
+    default: null
```

Comment:
Please add the following at the same indent level as `description`
`version_added: 2.2`